examples/platform/alerts_common_scenarios.py

```diff
@@ -113,6 +113,7 @@ def main():
     # Device - device - READ: For Device Searches
     # Alerts - org.alerts.close - EXECUTE:
     # Alerts - org.alerts.notes - CREATE, READ, UPDATE, DELETE
+    # Alerts - ThreatMetadata - org.xdr.metadata - READ
```

Comment:
This is not part of these changes, but I think for close alert another permission is needed `READ jobs.status`.

Reply:
yes, thank you